ksp_plugin_adapter/ksp_plugin_adapter_linux.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's not actually necessary any more, I'll remove it. I'd been hitting this bug but it's since been resolved: https://bugzilla.xamarin.com/show_bug.cgi?id=16791